packages/kit/src/loader/nuxt.ts

```diff
@@ -18,43 +18,58 @@ export interface LoadNuxtOptions extends LoadNuxtConfigOptions {
   config?: LoadNuxtConfigOptions['overrides']
 }
 
-export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
-  // Backward compatibility
-  opts.cwd = opts.cwd || opts.rootDir
-  opts.overrides = opts.overrides || opts.config || {}
-
-  // Apply dev as config override
-  opts.overrides.dev = !!opts.dev
-
+export async function importNuxtPackage<T = any> (rootDir = process.cwd()): Promise<{ version: 2 | 3, package: T }> {
   const nearestNuxtPkg = await Promise.all(['nuxt3', 'nuxt', 'nuxt-edge']
-    .map(pkg => resolvePackageJSON(pkg, { url: opts.cwd }).catch(() => null)))
+    .map(pkg => resolvePackageJSON(pkg, { url: rootDir }).catch(() => null)))
```

Comment:
Refactor: It is actually cwd not rootDir because of node_module resolution it can also look-up

Reply:
Also we might resolove `rootDir = resolve('.', rootDir)` for consistency once in first line of function